2_process/src/compare_to_historic.R

```diff
@@ -21,19 +25,23 @@ compare_to_historic <- function(target_name, historic_quantile_fn, current_data_
     
     site_current <- current_data %>% 
       filter(site_no == site) %>% 
+      # Add flag for whether an inverse of the GWL value should be used
+      mutate(is_inverse = site_no %in% inverse_sites) %>% 
       # Figure out the corresponding quantile for each daily value
       # If there are no non-NA quantiles available, return NA
       rowwise() %>% 
       mutate(daily_quant = ifelse(
         nrow(site_quantiles) > 0, 
-        yes = approx(x = site_quantiles$quantile_va, y = site_quantiles$quantile_nm, xout = GWL)$y,
+        yes = approx(x = site_quantiles$quantile_va, y = site_quantiles$quantile_nm, 
+                     xout = GWL*ifelse(is_inverse, -1, 1))$y,
```

Comment:
I think the ifelse logic to invert the values also needs to be done to `x = site_quantiles$quantile_va` so that the interpolation is applied to values of the same sign +/-

Reply:
With this change I am still missing a number of sites 
![image](https://user-images.githubusercontent.com/17803537/124539919-0456a000-dde4-11eb-9541-55bc4e4c86a1.png)

Reply:
It seems like you don't have the new historic data, which might be why there are so many `Quantiles not available for...returning NA` messages. I wonder if you can try:

```r
library(scipiper)
scmake("1_fetch/out/historic_gw_site_info_filtered.rds", force=TRUE)
scmake("1_fetch/out/historic_gw_quantiles.csv", force=TRUE)
```

If you get an AWS error, try 1) getting on VPN, and 2) updating `dssecrets` (pull from upstream then build the pkg) because the AWS key was recently updated by David.

Reply:
The quantiles are already reversed when they are originally generated (see [here](https://github.com/USGS-VIZLAB/gw-conditions/pull/24/files#diff-1c2392b3deb15657a887700756f213e59af7ba47bb0dbb7f659db96e00885d8eR17)), so I should just be inversing the groundwater levels being compared to them (_right?_)

Reply:
yes. This resolved the issue

Reply:
![image](https://user-images.githubusercontent.com/17803537/124639243-8df08700-de51-11eb-8d42-df30600ddabb.png)